src/stories/Spacing.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+type SizeProps = {
+  [key: string]: {
+    [key: string]: string;
+  };
+};
+
+export const SpacingBlocks = () => {
+  const sizes: SizeProps = {
+    xl: {
+      color: '--color-green-50',
+      mqBase: '32',
+      mqSmall: '48',
+      mqMedium: '64'
+    },
+    lg: {
+      color: '--color-blue-50',
+      mqBase: '16',
+      mqSmall: '24',
+      mqMedium: '32'
+    },
+    md: {
+      color: '--color-orange-50',
+      mqBase: '8',
+      mqSmall: '12',
+      mqMedium: '16'
+    },
+    sm: {
+      color: '--color-red-50',
+      mqBase: '6',
+      mqSmall: '8',
+      mqMedium: '8'
+    },
+    xs: {
+      color: '--color-amber-40',
+      mqBase: '4',
+      mqSmall: '4',
+      mqMedium: '4'
+    }
+  };
```

Comment:
This seems problematic in that it's statically declaring many values which may change if their equivalent design token values are change. 

- xs, sm, md, lg, xl are tied to the spacing variable names which could change in the design tokens
- the keys mqBase, mqSmall, mqMedium are tied to breakpoint variable names which may change in the same way
- the px values for the above keys are only correct so long as their equivalent values in the design tokens do not change

My guess is you already realise this, but you've created this object like so with the aim of marrying it to design tokens later?

Reply:
> My guess is you already realise this, but you've created this object like so with the aim of marrying it to design tokens later?

I do indeed realise this. All of these static token objects have been created as temporary placeholders and ultimately they will all be supplanted by files generated by Style Dictionary from design tokens.

I added a comment to try to illustrate this but it was in a [different PR](https://github.com/wellcometrust/design-system/pull/9/files#diff-fff70a23d29949151c2553e30e29031da44fdfd77fa626c3c8ac729d375a4fe4) so I've added it [here](https://github.com/wellcometrust/design-system/pull/8/files#diff-54a1dfac3e178d6ff58085ae0baaf62b28e1c0d8022d87b09a6ac075092b5101) as well

Reply:
Ace, all good - again just checking I was on the same page, and I figured you would already be on top of this stuff anyway!